keyboards/handwired/2x5keypad/keymaps/default/keymap.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include QMK_KEYBOARD_H
+
+#define WIN_TAB LGUI(KC_TAB)
+#define WIN_LOCK LGUI(KC_L)
+
+enum layers {
+  NORMAL_LAYER = 0,
+  MEDIA_LAYER,
+  TBD_LAYER2,
+  TBD_LAYER3
+};
+
+
+uint32_t layer_state_set_user(uint32_t state);
```

Comment:
This can be removed; QMK already declares this function.
The rest of this file should go in keymap.c instead - there is no need for the header.

Reply:
Will delete this (actually will delete keymap.h file).